app/ts/background/windows/interceptorAccess.ts

```diff
@@ -181,17 +174,17 @@ export async function requestAccessFromUser(
 			if (requests.findIndex((x) => x.accessRequestId === accessRequestId) === 0) {
 				await sendPopupMessageToOpenWindows({ method: 'popup_interceptorAccessDialog', data: requests })
 			}
-			return await sendPopupMessageToOpenWindows({ method: 'popup_interceptor_access_dialog_pending_changed', data: requests })
+			await sendPopupMessageToOpenWindows({ method: 'popup_interceptor_access_dialog_pending_changed', data: requests })
+			return await tryFocusingTabOrWindow({ type: openedDialog.popupOrTab.type === 'tab' ? 'tab' : 'window', id: openedDialog.popupOrTab.windowOrTab.id })
 		}
 		pendingAccessRequests.resolve(requests)
 	})
 }
 
 async function updateViewOrClose() {
 	const promises = await getPendingAccessRequests()
-	if (promises.length > 0) return sendPopupMessageToOpenWindows({ method: 'popup_update_access_dialog', data: promises })
+	if (promises.length > 0) return sendPopupMessageToOpenWindows({ method: 'popup_interceptorAccessDialog', data: promises })
```

Comment:
why the change from snake case to camel case on this method?

Reply:
the idea is to keep `popup_` prefix and then move to camelcase for all messages. As they are currently quite mixed. Not sue what is the best strategy with these 🤔